### DISCUSSION TITLE
What do we know about the hardware side of the problem?

### DISCUSSION THREAD

**Discussion by JamesPagetButler** [Open-Source-Software-Neighborhood-Watch/town-square#11] [General]
It seems that hardware and it's designs have a number challenges in terms of supply chain, are there good resources that have explored this side of the problem?

**Comment by tpike3** [comment]
I know there is a lot of work in supply networks right now. I have only looked at this tangentially but a few resources I am aware of that look at supply networks generally are: 

[Fewsion mapping America's Supply Chain](https://fewsion.us/) They also have summer education programs under resources.

[Fewsion Associated Nature paper](https://www.nature.com/articles/s41586-021-03621-0.epdf?sharing_token=HGVDjM5-KT3c10izUIjePdRgN0jAjWel9jnR3ZoTv0NOOu7KYlmdtXRGhoK8duzy29IgU53ApfG_jeCU95PE2b18bCCr7Xqks03TtHHYhiGN6deGIm_kUlHrqS_Tn3xcFx9gVuUnHzElJ246ty5zbDRBWw81d8dgj5_JkrInepo%3D)

[DARPA effort](https://www.grants.gov/web/grants/view-opportunity.html?oppId=338418)

**Reply by jspeed-meyers** [reply]
This is a good question. I'm glad @tpike3 has some answers. I don't! @JamesPagetButler, have you found any good writing or materials on hardware supply chain security?

**Comment by gps-iqt-labs** [comment]
Sorry -- just seeing this thread now. @JamesPagetButler, was there a specific HW segment you had in mind? µ-electronics or something else? 

Since the second part of your question explicitly asks about the challenges of hardware _design_, it's probably worth taking a look at Halvar Flake's work on "the anomaly of cheap complexity," which touches on an interesting range of related topics. These include: the macroscopic ramifications of Moore's Law and the unintended consequences of trying to reduce hardware + software product time to market through (over-)reliance on modular parts and modular SDKs. He also briefly gets into the parallels between software supply chain issues and hardware ones given how intertwined the two are becoming through transistor-heavy embedded systems. (Hope I'm not taking this discussion too far off-piste.)

- Slides here: https://rule11.tech/papers/2018-complexitysecuritysec-dullien.pdf
- Talk here: https://www.youtube.com/watch?v=q98foLaAfX8

**Reply by JamesPagetButler** [reply]
Wow, thanks for the resource you have given me a few nights of work.
